projects/knora-ui/src/lib/viewer/values/date-value/date-input/date-input.component.spec.ts

```diff
@@ -196,8 +194,6 @@ describe('DateInputComponent', () => {
 
   it('should show the toggle when not in readonly mode', () => {
```

Comment:
Could you change the title (just remove "when not in readonly mode")?

Reply:
> Could you remove the `readonly` `@Input`s from the wrapped components' classes (`projects/knora-ui/src/lib/viewer/values/color-value/color-picker/color-picker.component.ts` , `projects/knora-ui/src/lib/viewer/values/date-value/date-input/date-input.component.ts`,`projects/knora-ui/src/lib/viewer/values/time-value/time-input/time-input.component.ts`.)?

These have been left as readonly so that the user can't freely type in a value and must use the respective picker to ensure the value is in the correct format. I think it's better this way, especially for dates, but we can discuss it.

Reply:
fixed in a36ae2d